src/main/scala/it/pagopa/interop/agreementprocess/server/impl/Dependencies.scala

```diff
@@ -88,7 +92,8 @@ trait Dependencies {
         catalogManagement(blockingEc),
         tenantManagement(blockingEc),
         attributeRegistryManagement(blockingEc),
-        authorizationManagement(blockingEc)
+        authorizationManagement(blockingEc),
+        fileManager = fileManager(blockingEc)
```

Comment:
This is a `FeelFreeToHateMe™` comment. Can we use consistently named and non named parameters?